test/validation/api/classification/odp_classification_common.c

```diff
@@ -496,6 +508,15 @@ odp_packet_t create_packet(cls_packet_info_t pkt_info)
 			ODPH_ERR("odph_udp_tcp_chksum failed\n");
 			return ODP_PACKET_INVALID;
 		}
+	} else if (pkt_info.l4_type == CLS_PKT_L4_AH) {
+		ah->next_header = ODPH_IPV4;
+		ah->ah_len = ODPH_AHHDR_LEN;
```

Comment:
This is not quite correct (even though it would work here). The length value is the number of 32-bit words minus 2 and must be divisible by 2 with IPv6. I suggest setting ah_len to 4 so that an implementation that assumes correctly formatted AH packets does not get upset.